vita3k/config/src/config.cpp

```diff
@@ -249,6 +266,8 @@ void merge_configs(Config &lhs, const Config &rhs, const std::string &new_pref_p
         lhs.hardware_flip = rhs.hardware_flip;
     if (lhs.performance_overlay != rhs.performance_overlay && (!init || rhs.performance_overlay))
         lhs.performance_overlay = rhs.performance_overlay;
+    if (lhs.backend != rhs.backend && (!init || rhs.backend != -1))
```

Comment:
```suggestion
    if (lhs.backend != rhs.backend && (!init || rhs.backend != static_cast<int>(0)))
```
forget static cast and is 0 no -1

Reply:
No, that's wrong!

![5583755dba065d80218d0a3634c8d6195d6543d1v2_hq](https://user-images.githubusercontent.com/32211852/64339473-24cc2e00-cfb2-11e9-8cb4-092aa23b9a81.jpg)

This works, and changing it will break the config.yml file and overwrite it 100% of the time with an empty command line. I am sure of this. `-1` is the empty option, the config.yml file will never have it, but the command line might use it to say that you didn't specify the backend via `-B backend`.

If you want to specify another method to tackle this, go ahead- but changing this value to 0 will just create issues.

... sorry I've been playing too much danganronpa.

Reply:
hmm if you said ^^

Reply:
https://www.who.int/features/qa/gaming-disorder/en/